lib/concurrent.js

```diff
@@ -6,7 +6,7 @@ var BAIL_ERROR = new Error();
 module.exports = Concurrent;
 
 function Concurrent(tests, bail) {
-	if (!this instanceof Concurrent) {
+	if (!(this instanceof Concurrent)) {
```

Comment:
I'm surprised the linter didn't catch this. Would have thought ESLint would have rule for this.